packages/app/src/vis-packs/core/hooks.ts

```diff
@@ -99,58 +99,45 @@ export function useMappedArray<T extends unknown[] | TypedArray | undefined>(
   value: T,
   dims: number[],
   mapping: DimensionMapping,
-  autoScale?: boolean,
```

Comment:
It's as if `autoScale` was always true, now, so the `value` array always corresponds to the slice, never to the whole dataset. This opens the door to a bunch of simplifications, since it means we no longer need to 1) slice datasets in `applyMapping`, and 2) return a separate ndarray from `useMappedArray(s)` (which we needed to compute the domain of the entire dataset).

For now, I just remove the `autoScale` parameter. I will do more refactoring separately.